arrow/arrow.py

```diff
@@ -411,6 +415,10 @@ def interval(cls, frame, start, end, interval=1, tz=None):
         :param end: (optional) A datetime expression, the end of the range.
         :param interval: (optional) Time interval for the given time frame.
         :param tz: (optional) A timezone expression.  Defaults to UTC.
+        :param bounds: (optional) a ``str`` of either '()', '(]', '[)', or '[]' that specifies
+            whether to include or exclude the start and end values in the span. '(' excludes
```

Comment:
I think it needs to be interval instead of span in the docstring.

Reply:
Maybe "intervals" because it concerns every interval?

Reply:
Sounds good.